lib/Newsletter/Editor/StructureTransformer.php

```diff
@@ -49,14 +49,16 @@ private function transformTagsToBlocks($root, $image_full_width) {
           $image = $item;
         }
 
+        $width = $image->getAttribute('width');
+        $height = $image->getAttribute('height');
         return array(
           'type' => 'image',
           'link' => $link,
           'src' => $image->getAttribute('src'),
           'alt' => $image->getAttribute('alt'),
           'fullWidth' => $image_full_width,
-          'width' => $image->getAttribute('width'),
-          'height' => $image->getAttribute('height'),
+          'width' => $width === null ? 'auto' : $width,
```

Comment:
What happens in the Email editor if you add a post via a "Posts" block, and then attempt to access settings of the Image block?
In Image settings we have the input/slider to change the image width, currently it operates on pixels - I don't think it can handle `auto` value.